app/views/layouts/_topbar.html.haml

```diff
@@ -1,8 +1,8 @@
 %nav.top-bar{"data-topbar" => "", :role => "navigation"}
   %ul.title-area
     %li.name
-      %h1
-        %a{:href => "#"} Shostakodech
+      %h1#shostakodech
+        %a#home{:href => "#"} Shostakodech
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace